src/compliment/utils.clj

```diff
@@ -195,7 +195,10 @@
                             (not (.contains file "$")))]
              (.. (if (.startsWith file File/separator)
                    (.substring file 1) file)
-                 (replace ".class" "") (replace File/separator ".")))
+                 (replace ".class" "") (replace File/separator ".")
+                 ;; Address the issue #79 , on Windows, for prefix such
+                 ;; as "java.util.", the list of candidates was empty.
+                 (replace "/" ".")))
```

Comment:
Ok, so I now understand what you're doing: https://bitmingw.com/2020/05/16/do-not-use-file-separator-when-loading-java-resource-files/

In this case, looks like `(replace File/separator ".")` is redundant and should be removed.